components/motor/fake/motor.go

```diff
@@ -251,25 +251,25 @@ func (m *Motor) Direction() int {
 	return 0
 }
 
-// If revolutions is 0, the returned wait duration will be 0 representing that
-// the motor should run indefinitely.
 func goForMath(maxRPM, rpm, revolutions float64) (float64, time.Duration, float64) {
 	// need to do this so time is reasonable
 	if rpm > maxRPM {
 		rpm = maxRPM
 	} else if rpm < -1*maxRPM {
 		rpm = -1 * maxRPM
 	}
-	if rpm == 0 {
-		return 0, 0, revolutions / math.Abs(revolutions)
+
+	dir := 1.0
+	if rpm*revolutions == 0.0 {
+		dir = 0.0
+	} else if rpm*revolutions < 0.0 {
+		dir = -1.0
 	}
 
-	if revolutions == 0 {
-		powerPct := rpm / maxRPM
-		return powerPct, 0, 1
+	if rpm == 0 {
+		return 0, 0, dir
```

Comment:
let's make the checking functions helpers so we don't have to update them everywhere, we already have Errors and error helpers in motor.go, the check for 0 rpm and 0 revolutions is now the same., so we shouldn't copy/past identical code.

Reply:
This comment was not addressed. 

EDIT: I was not re-requested for review, you only committed, sorry! Was in a github PR review zone this morning.